app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -315,8 +315,14 @@ export default {
     },
     lastHeader: {
       immediate: true,
-      handler() {
-        if (this.session.signedIn) {
+      handler(newHeader) {
+        const waitTwentyBlocks = Number(newHeader.height) % 20 === 0
+        if (
+          this.session.signedIn &&
+          waitTwentyBlocks &&
+          this.$route.name === `validator` &&
+          this.myDelegation !== `--`
```

Comment:
```suggestion
          this.myDelegation !== 0
```
i don't think this will work anymore because of my validator-sorting PR.

Reply:
Hmm the function returns `---`, I'll switch it to `myBond`